src/did/connect/connect.js

```diff
@@ -77,53 +76,52 @@ export class DIDConnect {
   async #initiateWeb5Client() {  
     // Handler that will be used to step through the DIDConnect process phases
     const handleMessage = async (event) => {
-      const rpcMessage = parseJSON(event.data);
+      const rpcMessage = parseJson(event.data);
 
       switch (connectStep) {
 
-      case DIDConnectStep.Initiation: {
+      case DidConnectStep.Initiation: {
         // The Client App initiates the DIDConnect process, so no messages from the Provider are expected until the Verification step
         console.warn('Unexpected message received before Web5 Client was ready');
         break;
       }
 
-      case DIDConnectStep.Verification: {
+      case DidConnectStep.Verification: {
         const verificationResult = rpcMessage?.result;
         // Encrypted PIN challenge received from DIDConnect Provider
         if (verificationResult?.ok) {
           // Decrypt the PIN challenge payload
           const pinBytes = await this.web5.did.decrypt({
             did: this.#did.id,
             payload: verificationResult.payload,
-            privateKey: this.#did.keys[0].keypair.privateKeyJwk.d, // TODO: Remove once a keystore has been implemented
+            privateKey: this.#did.keys[0].keyPair.privateKeyJwk.d, // TODO: Remove once a keystore has been implemented
           });
-          const pin = this.web5.dwn.SDK.Encoder.bytesToString(pinBytes);
+          const pin = this.web5.dwn.sdk.Encoder.bytesToString(pinBytes);
 
           // Emit event notifying the DWA that the PIN can be displayed to the end user
           this.web5.dispatchEvent(new CustomEvent('challenge', { detail: { pin } }));
           
           // Advance DIDConnect to Delegation and wait for challenge response from DIDConect Provider
-          connectStep = DIDConnectStep.Delegation;
+          connectStep = DidConnectStep.Delegation;
           
           // Send queued PermissionsRequest to Provider.
-          this.#client.sendRequest(DIDConnectRPCMethods[connectStep], { message: this.#permissionsRequests.pop() });
+          this.#client.sendRequest(DidConnectRpcMethods[connectStep], { message: this.#permissionsRequests.pop() });
 
         } else {
           // TODO: Remove socket listeners, destroy socket, destroy this.#client, and emit error to notify user of app
         }
         break;
       }
 
-      case DIDConnectStep.Delegation: {
+      case DidConnectStep.Delegation: {
         const delegationResult = rpcMessage?.result;
 
         // Success
         if (delegationResult?.ok) {
           const authorizedDid = delegationResult?.message?.grantedBy;
-          // Register DID now that the connection was authorized
-          await this.web5.did.register({
+          // Set Managed DID now that the connection was authorized
+          await this.web5.did.manager.set(authorizedDid, {
```

Comment:
so a managed did is the one that is the "main" one for the DWN? do I have that right?

Reply:
A "managed" DID is one that the application/agent has been control of either directly (has access to keys or a signing function) or indirectly through an a remote application/agent.

All other DIDs are assumed to be controlled by other entities and are resolved in the normal manner.  Resolution varies depending on the DID method (`did:key`, `did:web`, `did:ion`).